### ISSUE TITLE
how to use custom dataloader with specific transformation to feed images to yolo v10

### ISSUE THREAD

**Issue by Khaled-Mohammed-Abdelgaber** [THU-MIG/yolov10#372]
i want to create a dataloader in pytorch and do some transformation on the image befor feeding it to the yolo model to increase model accuracy. and use yolo v10 as a part of bigger model ?
